StaticAnalyzer/views/shared_func.py

```diff
@@ -74,12 +74,16 @@ def unzip(app_path, ext_path):
         files = []
         with zipfile.ZipFile(app_path, "r") as zipptr:
             for fileinfo in zipptr.infolist():
+                filename = ''
                 filename = fileinfo.filename
                 if not isinstance(filename, str):
                     filename = str(
                         filename, encoding="utf-8", errors="replace")
                 files.append(filename)
-                zipptr.extract(fileinfo, str(ext_path))
+                full_filename = app_path + filename
+                full_filename = bytes(full_filename, 'utf-8')
+                full_filename = str(full_filename, encoding="utf-8", errors="replace")
+                zipptr.extract(full_filename, ext_path)

```

Comment:
Why do you need `full_filename` . Isn't `fileinfo.filename` enough?

Reply:
fileinfo.filename will contain xxxxx.apk
full_filename will contain /.root/MobSF/..../.../../xxxxx.apk

this PR need nore test before merging it , i need to double check a lot of things to ensure we don't have side effect  
delen app (md5: 1eccf8b457fb615039259c002ed7a17d) give me a lot of problem so i will investigate more